esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2328,6 +2328,24 @@ void CFileSprayEx::getDropZoneInfoByIP(double clientVersion, const char* ip, con
     }
 }
 
+static StringBuffer & expandLogicalAsPhysical(StringBuffer & target, const char * name, const char * separator)
```

Comment:
there is a similar function in dafdesc (makeSinglePhysicalPartName),
but it comes with baggage, e.g. it uses queryBaseDirectory().

So a new cleaner method like this is fine, should it go somewhere common like dautils.hpp ?